File: Design_Hashset/hashset.js
var MyHashSet = function () {
	this.payload = new Array(10000001).fill(false);
};

/**
 * @param {number} key
 * @returns {void}
 */
MyHashSet.prototype.add = function (key) {
	this.payload[key] = true;
};

/**
 * @param {number} key
 * @returns {void}
 */
MyHashSet.prototype.remove = function (key) {
	this.payload[key] = false;
};

/**
 * @param {number} key
 * @returns {boolean}
 */
MyHashSet.prototype.contains = function (key) {
	return this.payload[key];
};

/**
 * Your MyHashSet object will be instantiated and called as such:
 * var obj = new MyHashSet()
 * obj.add(key)
 * obj.remove(key)
 * var param_3 = obj.contains(key)
 */

File: Design_Hashset/hashset.cs
public class MyHashSet
{
    private bool[] payload;

    public MyHashSet()
    {
        payload = new bool[1000001];
    }

    public void Add(int key)
    {
        payload[key] = true;
    }

    public void Remove(int key)
    {
        payload[key] = false;
    }

    public bool Contains(int key)
    {
        return payload[key];
    }
}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet obj = new MyHashSet();
 * obj.Add(key);
 * obj.Remove(key);
 * bool param_3 = obj.Contains(key);
 */
